docs/src/components/Document.js

```diff
@@ -40,11 +40,9 @@ const Document = ({ Body, children, Head, Html, siteData: { dev, versions } }) =
         }.js`}
       />
       <script
-        src={
-          dev
-            ? ` https://cdn.jsdelivr.net/npm/@hot-loader/react-dom@${versions.react}/umd/react-dom.development.js`
```

Comment:
There is no `@hot-loader/react-dom` for 17.0.1. It also creates weird issues with HMR...